pkg/controlplane/instance.go

```diff
@@ -707,10 +666,18 @@ func (m *Instance) InstallAPIs(apiResourceConfigSource serverstorage.APIResource
 			m.GenericAPIServer.AddPostStartHookOrDie(name, hook)
```

Comment:
this will run the repair loops for the ClusterIPs and nodePorts, right?

Reply:
yes, as wired in storage_core.go